redaxo/src/core/update.php

```diff
@@ -7,13 +7,21 @@
     throw new rex_functional_exception(rex_i18n::msg('setup_301', PHP_VERSION, '7.1.3'));
 }
 
-$mysqlVersion = rex_sql::getServerVersion();
-$minMysqlVersion = '5.5.3';
-if (rex_string::versionCompare($mysqlVersion, $minMysqlVersion, '<')) {
-    // The message was added in REDAXO 5.6.0, so it does not exist while updating from previous versions
-    $message = rex_i18n::hasMsg('sql_database_min_version')
-        ? rex_i18n::msg('sql_database_min_version', $mysqlVersion, $minMysqlVersion)
-        : "The MySQL version $mysqlVersion is too old, you need at least version $minMysqlVersion!";
+$minMysqlVersion = '5.6';
```

Comment:
kommentar dass wir hier die konstanten nicht nehmen können, weil..

Reply:
Das steht schon ganz oben in der datei